srcalc/src/main/java/gov/va/med/srcalc/domain/calculation/PopulatedDisplayGroup.java

```diff
@@ -0,0 +1,204 @@
+package gov.va.med.srcalc.domain.calculation;
+
+import gov.va.med.srcalc.domain.Patient;
+import gov.va.med.srcalc.domain.model.BooleanVariable;
+import gov.va.med.srcalc.domain.model.DiscreteNumericalVariable;
+import gov.va.med.srcalc.domain.model.MultiSelectVariable;
+import gov.va.med.srcalc.domain.model.NumericalVariable;
+import gov.va.med.srcalc.domain.model.ProcedureVariable;
+import gov.va.med.srcalc.domain.model.Variable;
+import gov.va.med.srcalc.domain.model.VariableGroup;
+import gov.va.med.srcalc.vista.HealthFactor;
+import gov.va.med.srcalc.web.view.BooleanVariableView;
```

Comment:
Like my previous comment, the domain model should not depend on other integration packages. I think this class should move to `web.view`.